packages/flutter_tools/test/commands.shard/hermetic/analyze_continuously_test.dart

```diff
@@ -60,7 +60,7 @@ void main() {
     pubspecFile.writeAsStringSync('''
   name: foo_project
   environment:
-    sdk: '>=2.10.0 <4.0.0'
+    sdk: '>=2.12.0 <4.0.0'
```

Comment:
This LGTM, just verifying there are no other hard-coded constraints we need to bump

Reply:
Feel free to approve and tag for autosubmit if you find that this is a sufficiently complete fix.